<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pinhole_mask_dialog</class>
 <widget class="QDialog" name="pinhole_mask_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>152</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Create Pinhole Mask</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="pinhole_diameter_label">
     <property name="text">
      <string>Pinhole Diameter:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="ScientificDoubleSpinBox" name="pinhole_diameter">
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="suffix">
      <string> μm</string>
     </property>
     <property name="decimals">
      <number>8</number>
     </property>
     <property name="maximum">
      <double>100000000.000000000000000</double>
     </property>
     <property name="value">
      <double>150.000000000000000</double>
     </property>
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="pinhole_thickness_label">
     <property name="text">
      <string>Pinhole Thickness:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="ScientificDoubleSpinBox" name="pinhole_thickness">
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="suffix">
      <string> μm</string>
     </property>
     <property name="decimals">
      <number>8</number>
     </property>
     <property name="maximum">
      <double>100000000.000000000000000</double>
     </property>
     <property name="value">
      <double>75.000000000000000</double>
     </property>
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>The pinhole mask may be deleted in the mask manager.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScientificDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>scientificspinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pinhole_diameter</tabstop>
  <tabstop>pinhole_thickness</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>pinhole_mask_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>pinhole_mask_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
